src/components/inputs/notes.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <div>
+        <textarea            
+            v-model="newNote"           
+            :id="$attrs.id"
+            :class="$attrs.class"
+            :required="$attrs.required" 
+            :placeholder="$attrs.placeholder"
+            :disabled="$attrs.disabled"
+            :rows="$attrs.rows" 
+            :cols="$attrs.cols" 
+            @change="save" >
+        </textarea>       
+   
+        <table class="table table-striped" v-if="notes?.length">
+            <tr v-for="(item, index) in notes" :key="index">
+                <td>{{item}}  </td>   
+                <td style="width:10px;"><span class="btn" @click="remove(index)"><i class="fa fa-trash-o icon"></i></span></td>
+            </tr>
+        </table>   
+    </div> 
+</template>
+
+<script setup>
+   import { ref, computed } from 'vue'
+   import {useUser} from '../../services/composables/useAuth'
+  
+   let lastNote = '';
+   const newNote = ref();
+   const user = useUser();
+
+   const model = defineModel({type:String, required:true});
+   
+   let notes = computed(()=>{
+        if(model.value)
+            return  (JSON.parse(model.value)).filter(e=>e!=lastNote);
+        return [];
+   }); 
+   
+   const remove=(index)=>{  
+        for (var i = 0; i < notes.value.length; i++) {        
+            if(notes.value[i] == notes.value[index]){             
+                notes.value.splice(i, 1);        
+            }
+        }  
```

Comment:
`notes.value.splice(index, 1);` this should be enough

Reply:
done